### PR TITLE
Update the stacks.toml with new 2022 repositories

### PR DESCRIPTION
**Pull request by jdbender66**
Collected an updated list of Github repositories related to Stacks development

### CODE REVIEW

File: data/ecosystems/s/stacks.toml [hunk 1]
```diff
@@ -29,6 +29,16 @@ github_organizations = [
...
   "https://github.com/StackedSats",
+  "https://github.com/Trust-Machines",
+  "https://github.com/secretkeylabs",
+  "https://github.com/Clarity-Innovation-Lab",
+  "https://github.com/fungible-systems",
+  "https://github.com/mechanismHQ",
+  "https://github.com/stacksfoundation",
+  "https://github.com/magicstx",
+  "https://github.com/lnswap",
+  "https://github.com/layerswap",
```
Comment (jubos) [inline data/ecosystems/s/stacks.toml:40]: Most of these look like Stacks orgs except for layerswap.  Can you remove this one?

Also if possible, many of these orgs look like they could be their sub ecosystems of Stacks or Bitcoin.  If they have a discord / twitter account, that usually is a good heuristic of whether they should be their own ecosystem.

**Review by jubos — CHANGES_REQUESTED** [review]

File: data/ecosystems/s/stacks.toml (hunk 1 above)
Comment (jdbender66) [reply data/ecosystems/s/stacks.toml]: Removed Layerswap! Appreciate the fast reviews.

Comment (diwakergupta) [reply data/ecosystems/s/stacks.toml]: @jubos any further feedback? Any blockers to merge this?

**Comment by jubos** [comment]
Looks good, thanks for the updates!  Merged

**Pull request merged**